coffee_core/src/coffee.rs

```diff
@@ -573,6 +587,7 @@ impl PluginManager for CoffeeManager {
                 .get_mut(repo_name)
                 .ok_or_else(|| error!("repository with name: {repo_name} not found"))?;
 
+            repo.change_root_path(&self.config.path());
```

Comment:
When we change the root path of the repository, are plugins' path also modified to reflect this change?

Reply:
I think no, and we sould too right?

Reply:
Yes we should

We have 2 options to handle this change (moving repositories folder from being global to being network specific including plugins' paths information):
- Handle everything in the nurse strategy patch method
- Implement a method to `repository` that handles it (similar to `change_root_path` in this PR)

I prefer the first option since I don't really see us using this method anywhere else in the code

Reply:
just reindexing the repository with `add_remote` did the job in a smart way